.github/workflows/examples.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       run: |
         source ./dev/install-common-deps.sh
         source activate test-environment
-        pytest --verbose tests/examples --large
+        pytest --verbose tests/examples --durations=30 --large
```

Comment:
This overwrites `--durations` option specifed in `pytest.ini`.

Reply:
We have currently 22 example tests so 30 should be sufficient for a while.

Reply:
Is there a way to show durations for all tests by default? This way we do not need to actively manage this number as count of test increases?

Reply:
We could set `durations` to `0` which shows everything including setup and teardown.